tests/app/celery/test_letters_pdf_tasks.py

```diff
@@ -28,25 +28,6 @@ def test_should_have_decorated_tasks_functions():
     assert create_letters_pdf.__wrapped__.__name__ == 'create_letters_pdf'
 
 
-@pytest.mark.parametrize('crown_flag,expected_crown_text', [
```

Comment:
Why were these deleted? They seem like valid tests (may need updating to determine if it was a test or not test though)

Reply:
They weren't deleted, just moved to a more logical place to `test_letter_utils.py`